src/EnergyPlus/SimulationManager.cc

```diff
@@ -1190,6 +1190,14 @@ namespace SimulationManager {
             }
         }
 
+        ObjexxFCL::gio::write(OutputFileInits, fmtA) << "! <PerformancePrecisionTradeoffs:DirectSolution>, Simulation {{Yes/No}}, Direct Solution";
```

Comment:
@lgu1234 I was expecting the output to look like this, similar to Simulation Control:
```
! <Performance Precision Tradeoffs>, Use Coil Direct Simulation, Zone Radiant Exchange Algorithm
Performance Precision Tradeoffs, Yes, ScriptF
```
Also, we're no longer using gio::write - see elsewhere in the same source file (see #7746).

And this should really be printed later in the file, I would put it right after SimulationControl.

Reply:
@mjwitte I am going to make changes to "print(outputFiles.eio...". Since I am not familiar with Zone Radiant Exchange Algorithm, I will write Use Coil Direct Simulation first and let other people add more options.